ipywidgets/widgets/widget.py

```diff
@@ -378,14 +378,14 @@ def _allow_write(self):
     @contextmanager
     def hold_sync(self):
         """Hold syncing any state until the context manager is released"""
```

Comment:
Can we change this description to "until the outermost context manager exits" to give some clue to the code below and more precisely document what is going on?